docs/evm.logic.rst

```diff
@@ -0,0 +1,134 @@
+evm\.logic package
```

Comment:
Yes, all autodoc files, not intended to be merged just  from me trying to get my install up and working and interacting with the code base. Based on #19 I'm going to just kill these entirely, they are outside spec.